package.json

```diff
@@ -13,7 +13,9 @@
   "dependencies": {
     "@nuxt/content": "^1.14.0",
     "core-js": "^3.15.1",
-    "nuxt": "^2.15.7"
+    "nuxt": "^2.15.7",
+    "remark-directive": "^1.0.1",
```

Comment:
Maybe we also need a dependency list for those pro users, especially given the fancy handling you made with the portable format of these stories. We can make a developer readme later and list the packages used to handle these markdown files in such an elegant way (e.g. remark-directive, remark-sectionize), with some further explanation and links to the detailed documentation.

Reply:
You mean a bit of explanation for how it works? I wanted to write some of those things in the blog post I'm working on. Perhaps we can reuse it.